### ISSUE TITLE
parseWKT Segmentation fault

### ISSUE THREAD

**Issue by OmriRoames** [JuliaGeo/LibGEOS.jl#22]
parseWKT is segfauling when it is given a polygon that does not close.

```julia
p2 = parseWKT("POLYGON((0.0 0.0,1.0 0.0,1.0 1.0,0.0 1.0))")
```

**Comment by yeesian** [comment]
Thanks! We should handle errors similar to how shapely does it:

```python
>>> from shapely.wkt import loads
>>> loads("POLYGON((0.0 0.0,1.0 0.0,1.0 1.0,0.0 1.0))")
IllegalArgumentException: Points of LinearRing do not form a closed linestring
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/yeesian/anaconda2/lib/python2.7/site-packages/shapely/wkt.py", line 10, in loads
    return geos.WKTReader(geos.lgeos).read(data)
  File "/Users/yeesian/anaconda2/lib/python2.7/site-packages/shapely/geos.py", line 215, in read
    raise ReadingError("Could not create geometry because of errors "
shapely.geos.ReadingError: Could not create geometry because of errors while reading input.
```

We should address this in #21. PRs welcome!

**Comment by yeesian** [comment]
Closed in favour of #21.
